src/SprykerEco/Zed/AfterPay/Business/Payment/PaymentWriter.php

```diff
@@ -40,6 +40,20 @@ public function setIdReservationByIdSalesOrder(string $idReservation, int $idSal
             ->save();
     }
 
+    /**
+     * @param string $customerNumber
+     * @param int $idSalesOrder
+     *
+     * @return void
+     */
+    public function setCustomerNumberByIdSalesOrder(string $customerNumber, int $idSalesOrder): void
```

Comment:
Could you make it via entity manager?

Reply:
we don't have entity manager at all in this integration